src/libcore/num/mod.rs

```diff
@@ -1021,6 +1102,34 @@ $EndFeature, "
             }
         }
 
+        doc_comment! {
+            concat!("Wrapping Euclidean modulo. Computes `self.mod_euc(rhs)`, wrapping around at the
+boundary of the type.
+
+Such wrap-around never actually occurs mathematically; implementation artifacts make `x % y`
```

Comment:
Same as above. How about:

_Wrapping will occur in `MIN % -1` on a signed type (where `MIN` is the negative minimal value for the type). In this case, this method return 0._